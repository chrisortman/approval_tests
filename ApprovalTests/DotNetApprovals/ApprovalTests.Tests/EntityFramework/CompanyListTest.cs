using System.Collections.Generic;
using ApprovalTests.Reporters;
using ApprovalUtilities.Persistence;
using NUnit.Framework;
using ApprovalUtilities.Persistence.EntityFramework;

namespace ApprovalTests.Tests.EntityFramework
{
	[TestFixture]
	//[UseReporter(typeof(FileLauncherReporter))]
	public class CompanyListTest
	{
		[Test]
		public void TestRoster()
		{
			var mockLoader = new MockLoader<IEnumerable<Company>>( new []{new Company(){Name = "Motorola"}});
			var html = CompanyList.GetCompanyRoster("M", mockLoader);
			ApprovalTests.Html.Approvals.ApproveHtml(html);
		}

		[Test]
		public void TestLoader()
		{
			Approvals.Approve(CompanyList.CompanyNameLoader("Mi").Singleton());
		}
	}
}