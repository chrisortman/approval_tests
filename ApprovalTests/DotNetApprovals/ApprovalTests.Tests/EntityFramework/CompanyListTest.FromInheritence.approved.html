<html>
  <body>
    <li>Spun</li>
  </body>
</html>